README.md

```diff
@@ -10,6 +10,54 @@ PAID, also known as "Pflegeabrechnung in Deutschland", is a library written in t
 npm install paid-care
 ```
 
+## Usage
+
+### 1. Get data
+
+First we need to download some data: 
+- Institution data (who pays the receipts, where to send the receipts to and how, to whom to encrypt the receipts, ...)
+- Public keys of institutions to encrypt to
+- Hilfsmittelverzeichnis (catalogue of invoicable therapeutic appliances)
+
+The data could potentially change daily but should at least be fetched once a quarter of a year.
+
+```typescript
+import fetchPublicKeyInfos from "pki/fetcher"
+import fetchInstitutions from "kostentraeger/fetcher"
+
+const publicKeyInfos = await fetchPublicKeyInfos()
+const institutionsParseResult = await fetchInstitutions()
+const institutionsList = institutionsParseResult.map(it => it.institutionsList)
+```
+
+### 2. Stringify, Revive data
+
+TODO: maybe this could all be under the hood: 
+1. Just one fetchData() which fetches everything in a "giant" JSON
+2. the main class of the library could accept that data optionally, if it is not supplied, it will
+   fallback to the data shipped in the dist so a "zero conf" is possible
+
+This data should be persisted in a file or `localStorage` because the download is quite large and
+the parsing a bit laborious because there are many validity checks. The data can be stringified
+to JSON and revived like this:
+
+```typescript
+import { institutionListReplacer, institutionListReviver } from "kostentraeger/types"
+import { ikPublicKeyInfoReplacer, ikPublicKeyInfoReviver } from "pki/types"
+
+// stringify the data (for persisting it somewhere)
+const pkInfosString = JSON.stringify(publicKeyInfos, ikPublicKeyInfoReplacer)
+const institutionsListString = JSON.stringify(institutionsList, institutionListReplacer)
+
+// revive the data
+const publicKeyInfos2 = JSON.parse(pkInfosString, ikPublicKeyInfoReviver)
+const institutionsList2 = JSON.parse(institutionsListString, institutionListReviver)
```

Comment:
What do think about hiding the replacer / reviver implementation detail? It's a minor change, maybe like this:

```suggestion
import { serializeInstitutionsList, deserializeInstitutionsList } from "kostentraeger/types"
import { serializePublicKeyInfos, deserializePublicKeyInfos } from "pki/types"

// stringify the data (for persisting it somewhere)
const pkInfosString = serializePublicKeyInfos(publicKeyInfos)
const institutionsListString = serializeInstitutionsList(institutionsList)

// revive the data
const publicKeyInfos2 = deserializePublicKeyInfos(pkInfosString)
const institutionsList2 = deserializeInstitutionsList(institutionsListString)
```

Reply:
Both this data is basically only used in conjunction with each other. So to make the interface more slick here, it should be merged into one data structure anyway, i.e. 

`const institutionData = deserializeInstitutionData(string)`

Regarding the replacer/reviver, I don't mind either way. I just find it a bit weird that the data class would then dictate into which format it is serialized.

Reply:
Good, if the interface could be even simpler, I am in!

Reply:
I'd defer that to a separate PR though